Studs/Events/ViewModels/EventCellViewModel.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  EventCellViewModel.swift
+//  Studs
+//
+//  Created by Willy Liu on 2019-02-22.
+//  Copyright © 2019 Studieresan. All rights reserved.
+//
+
+import Foundation
+
+struct EventCellViewModel {
+    let month: String
+    let day: String
+    let companyName: String?
```

Comment:
Can a company not have a name?

Reply:
Right good point. The `Event` struct looks like this:

```swift
struct Event: Decodable {
    let id: String
    let companyName: String?
    // ...
```

So if I set `companyName` as non-optional, line 108 in EventsViewModel would give me a warning:
```swift
let viewModel = EventCellViewModel(month: month, day: day, companyName: event.companyName)
```

Reply:
It should probably not be null there either then.